public/app/stores/configureStore.ts

```diff
@@ -0,0 +1,17 @@
+import { createStore, applyMiddleware, compose, combineReducers } from 'redux';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import sharedReducers from 'app/core/reducers';
+import alertingReducers from 'app/features/alerting/state/reducers';
```

Comment:
For the future it can be nice to have a configureFeatures or similar in 'app/features/<something>` so that you don't need to jump to this file each time you've added a redcuer.

Each feature may have an index.ts file that exports reducers according to agreed schema and someone else decide where they should live in the state tree. Guess each feature will be like a module that expert certain stuff that can be used for bootstrapping.

Reply:
> For the future it can be nice to have a configureFeatures or similar in 'app/features/` so that you don't need to jump to this file each time you've added a redcuer.

Maybe but would that just mean you have to jump to a different file to import it, basically the same action but just a different file? 

I mean the same model that you describe is used by core/reducers (index file that brings in other reducers)

Reply:
Yeah your right. Disregard my comment, this is probably something that will mature when we get more and more features/pages in react.

Reply:
the best thing would be to have self registering reducers, but not seen any example of that :) or a generic import rule pattern import * as reducers from 'app/features/*/reducers'   :)